include/frozen/bits/pmh.h

```diff
@@ -186,13 +196,18 @@ pmh_tables<M, Hash> constexpr make_pmh_tables(const carray<Item, N> &
   // Step 2: Sort the buckets to process the ones with the most items first.
   auto buckets = step_one.get_sorted_buckets();
 
+  // Special value for unused slots. This is purposefully the index
+  // one-past-the-end of 'items' to function as a sentinal value. Both to avoid
```

Comment:
typo: sentinal -> sentinel. I <3 the trick here.

Reply:
Thanks. I really need to keep spell checking enabled for comments ;-)

I just realized this not needing to call `KeyEqual` has another advantage: one less branch, for people that care about SPECTRE, to worry about.